src/test/kotlin/com/amazon/opendistroforelasticsearch/indexmanagement/indexstatemanagement/coordinator/ManagedIndexCoordinatorIT.kt

```diff
@@ -69,24 +56,7 @@ class ManagedIndexCoordinatorIT : IndexStateManagementRestTestCase() {
         }
     }
 
-    fun `test creating index with invalid policy_id`() {
-        wipeAllODFEIndices()
-        assertIndexDoesNotExist(INDEX_MANAGEMENT_INDEX)
-
-        val indexOne = randomAlphaOfLength(10).toLowerCase(Locale.ROOT)
-        val indexTwo = randomAlphaOfLength(10).toLowerCase(Locale.ROOT)
-        val indexThree = randomAlphaOfLength(10).toLowerCase(Locale.ROOT)
-
-        createIndex(indexOne, Settings.builder().put(ManagedIndexSettings.POLICY_ID.key, " ").build())
-        createIndex(indexTwo, Settings.builder().put(ManagedIndexSettings.POLICY_ID.key, "").build())
-        createIndex(indexThree, Settings.builder().putNull(ManagedIndexSettings.POLICY_ID.key).build())
-
-        waitFor {
-            assertFalse("ISM index created for invalid policies", indexExists(INDEX_MANAGEMENT_INDEX))
-        }
-    }
-
-    fun `test deleting index with policy_id`() {
+    fun `test deleting index will remove managed-index`() {
         val (index) = createIndex(policyID = "some_policy")
```

Comment:
With the changes in this PR, since won't be using the index setting to add to policy anymore there might need to be changes made to `createIndex` as it takes a `policyID` and creates an index with `ManagedIndexSettings.POLICY_ID` being that `policyID`

Reply:
yes, modified the createIndex function, it will first create index, then if there is policyID parameter exist, it will do a add policy call

Reply:
Ah, my mistake, I somehow missed that